src/trajectory/Interpolated.cpp

```diff
@@ -152,6 +164,16 @@ const statespace::StateSpace::State* Interpolated::getWaypoint(
 }
 
 //=============================================================================
+double Interpolated::getWaypointTime(size_t _index) const
+{
+  if (_index < mWaypoints.size())
+    //return getStartTime() + mWaypoints[_index].t;
+    return mWaypoints[_index].t;
```

Comment:
In the Interpolated class, getStartTime() is taken as the time of the first way point.
"mWaypoints.front().t"
I will double check the correctness.